ironfish/src/primitives/target.test.ts

```diff
@@ -121,11 +121,29 @@ describe('Calculate target', () => {
 
       expect(newTarget.targetValue).toBeLessThan(target.targetValue)
     }
+
+    // for any time 35-45 seconds after the last block, difficulty should increase by previous block's difficulty / BigInt(2048) * 2
+    for (let i = 35; i < 45; i++) {
+      const time = new Date(now.getTime() + i * 1000)
+
+      const difficulty = BigInt(231072)
+      const target = Target.fromDifficulty(difficulty)
+
+      const diffInDifficulty = (difficulty / BigInt(2048)) * BigInt(2)
+
+      const newDifficulty = Target.calculateDifficulty(time, now, difficulty)
+      const newTarget = Target.calculateTarget(time, now, target)
+
+      expect(newDifficulty).toBeGreaterThan(difficulty)
+      expect(BigInt(difficulty) + diffInDifficulty).toEqual(newDifficulty)
+
+      expect(newTarget.targetValue).toBeLessThan(target.targetValue)
+    }
   })
 
-  it('keeps difficulty/target of parent block header if time differnece is between 10 and 20 seconds', () => {
+  it('keeps difficulty/target of parent block header if time differnece is between 55 and 64 seconds', () => {
```

Comment:
you mean 65 seconds?

Reply:
The seconds when the difficulty is not changing:
55 56 57 58 59 60 61 62 63 64 -> 10 numbers

Reply:
sure but the for loop is exclusive. 
small nit, doesn't really matter, but the comment isn't really matching the code then